CMakeLists.txt

```diff
@@ -0,0 +1,83 @@
+cmake_minimum_required(VERSION 3.1)
+
+project(http-parser VERSION 2.8.0)
+
+add_library(http_parser "http_parser.c" "http_parser.h")
+if(CMAKE_BUILD_TYPE STREQUAL "Debug")
```

Comment:
Will not work for multi-config generators like Xcode. Generator expression should be used here, please refactor it into something like this:
* https://github.com/forexample/package-example/blob/8b4b91bc6f3bc61de958e61536f63fd35a2648a3/Foo/CMakeLists.txt#L33-L39

Reply:
Good catch will do.